dask_ml/preprocessing/_block_transformer.py

```diff
@@ -60,15 +64,19 @@ class BlockTransformer(BaseEstimator, TransformerMixin):
     Dask Name: hash_pandas_object, 60 tasks
     """
 
-    def __init__(self, func, *, validate=False, **kw_args):
+    def __init__(self, func: Callable, *, validate: bool = False, **kw_args: Any):
```

Comment:
Is there any way to restrict the type on `Callable`? IIUC, we only know that the first argument will be a NumPy Array or Pandas DataFrame, and that it should return one of those, correct?

Reply:
Yeah.

I have been able to restrict the return type on `Callable`.